config/locales/en.yml

```diff
@@ -19,6 +19,13 @@ en:
       not_confirmed: "A confirmation email was sent to your account at '%{email}'. You must follow the instructions in the email before your account can be activated"
     confirmations:
       invalid_token: "Invalid confirmation token. Please try again"
+      user_not_found: "Unable to find user with email '%{email}'."
```

Comment:
Isn't this the only one you need? Why add the others?

Reply:
In a previous comment I left, I had mentioned that a few of these were copied over from Devise and DTA. I had asked for a consensus on which one (or ones) you would like. If you notice, `send_instructions` and `send_paranoid_instructions` both come from Devise so I figured that maybe a discussion one which one we wanted but also supporting a "paranoid" version might be advantageous.

Let me know what you think about that and I'll make changes accordingly.

Reply:
Also, if you're referring to the `confirmation_instructions:` section, those needed to be included in the locale for the mailer views.

Reply:
please just add the ones your PR uses. Others are not used in our templates or inherited from Devise.

Reply:
Ok got it!

Reply:
If we are going to depend on Devise fall back locales, then I should probably remove the `send_instructions` locale and use Devise's version since that's where I got this. Also, we didn't have `.confirm_link_msg` or `.confirm_account_link` so I added those. They didn't come from `devise.en.yml` so my guess is you were referring to DTA's `en.yml`? I'm keeping them here unless you want to fallback on DTA's locales as well as Devise.